<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="imageLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionOpen_O"/>
    <addaction name="actionSave_S"/>
    <addaction name="actionSave_as_A"/>
    <addaction name="separator"/>
    <addaction name="actionQuit_Q"/>
   </widget>
   <widget class="QMenu" name="menuCryptor_P">
    <property name="title">
     <string>Cryptor(&amp;P)</string>
    </property>
    <addaction name="actionEncryptor_E"/>
    <addaction name="actionDecryptor_D"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuCryptor_P"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen_O">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave_S">
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
  </action>
  <action name="actionSave_as_A">
   <property name="text">
    <string>Save as(&amp;A)</string>
   </property>
  </action>
  <action name="actionQuit_Q">
   <property name="text">
    <string>Quit(&amp;Q)</string>
   </property>
  </action>
  <action name="actionEncryptor_E">
   <property name="text">
    <string>Encryptor(&amp;E)</string>
   </property>
  </action>
  <action name="actionDecryptor_D">
   <property name="text">
    <string>Decryptor(&amp;D)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
